client/lib/abtest/active-tests.js

```diff
@@ -101,5 +101,14 @@ module.exports = {
 			description: 50
 		},
 		defaultVariation: 'description'
-	}
+	},
+	headstart: {
+		datestamp: '20160203',
+		variations: {
+			original: 20,
```

Comment:
sounds good to me. I'd even be fine with fewer than 20%, but I suppose we need some data.